gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyDedupReducer.java

```diff
@@ -61,6 +65,14 @@ protected void reduce(OrcKey key, Iterable<OrcValue> values, Context context)
     int valueHash = 0;
 
     for (OrcValue value : values) {
+      if (recordCounter == 1) {
+        log.info("Reduced first value");
+      }
+      recordCounter++;
+      if (recordCounter % 1000 == 0) {
+        log.info("Reduced %s values for key %s", recordCounter, key);
```

Comment:
Logger uses `{}` instead of %s. %s is for string.format

Reply:
(Reduced %s values for key %s", recordCounter, key) -> "(Reduced %s values till now)" ?